openbb_terminal/API_README.md

```diff
@@ -1,84 +1,278 @@
+
+## Setup
+### 1. Import OpenBB SDK
+
+First off, import OpenBB SDK into your python script or Jupyter Notebook with:
+
+```
 from openbb_terminal.api import openbb
 ```
 
-This imports all the commands at once. Now you can directly begin to use it. The api function is structured
-in the way that it always retrieves the underlying data. For charts add the `chart=True` parameter.
+This imports all Terminal commands at once. To see all the available commands, you can press `tab` in jupyter notebook.
+Another approach is to check out [OpenBB SDK Documentation](https://openbb-finance.github.io/OpenBBTerminal/api/), where you can explore its capabilities
+
+
+### 2. Customize chart style
+With OpenBB SDK, you can customize your chart style. You can switch between `dark` and `light` easily using this block of code:
 
-For example see:
 
-```python
-# Returns data:
-openbb.stocks.ba.snews("aapl")
-# Returns charts:
-openbb.stocks.ba.snews("aapl", chart=True)
 ```
+from openbb_terminal.api import TerminalStyle
+theme = TerminalStyle("light", "light", "light")
 
-This allows easy integration to the jupyter notebook and allows you to build new applications on top of the terminal.
-The api also has new functionalities that are used in the backend of the Terminal (CLI). With these backend functions
-you can develop new functionalities and avoid copy-pasting the code from the repository.
+```
 
-**Note:**\
-API keys for data fetching still have to be entered
+<img width="813" alt="Screenshot 2022-10-03 at 23 56 52" src="https://user-images.githubusercontent.com/40023817/193700307-cbb12edc-0a5d-4804-9f3c-a798efd9e69d.png">
 
-### Jupyter Notebook Tricks
+OR
 
-**Get  matplotlib charts in the output cells**\
-If you copy-paste the code below and use it as your initialization then you're matplotlib graphs will be inside
-the result cell.
+```
+from openbb_terminal.api import TerminalStyle
+theme = TerminalStyle("dark", "dark", "dark")
 
-```python
-import matplotlib.pyplot as plt
-import matplotlib_inline.backend_inline
-from openbb_terminal.api import openbb
-%matplotlib inline
-matplotlib_inline.backend_inline.set_matplotlib_formats("svg")
 ```
 
-**Get function signature and docstring**\
-When you press `shift + tab` in jupyter notebook while having the mouse parser in an API function, you get the
-signature and docstring of the function.
 
-### Visual Studio Code Tricks
+<img width="791" alt="Screenshot 2022-10-03 at 23 46 33" src="https://user-images.githubusercontent.com/40023817/193699221-e154995b-653c-40fd-8fc6-a3f8d39638db.png">
+
 
-**Get function docstring and signature**\
-In order to get the docstrings and function signatures for the API when opening a Jupyter Notebook in VSCode,
-you have to install the [Jupyter PowerToys
+
+### 3. Access Documentation
+Each and every command of OpenBB SDK has detailed documentation about input parameters and returned outputs. You can access them in multiple ways:
```

Comment:
Why do we need to demonstrate three methods of showing help? Can't you just do a bullet list showing the options and highlighting our own documentation (from openbb.co/docs) through a screenshot?